src/Console/ConfigurationResolver.php

```diff
@@ -401,52 +403,69 @@ private function resolveConfigPath()
             return;
         }
 
-        if (null === $this->path) {
+        if (empty($this->path)) {
```

Comment:
`$this->path` could be an empty array when no `path` argument was passed via CLI